src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/impl/CreditorsWithinOneYearTransformerImpl.java

```diff
@@ -0,0 +1,245 @@
+package uk.gov.companieshouse.web.accounts.transformer.smallfull.impl;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.accounts.smallfull.creditorswithinoneyear.CreditorsWithinOneYearApi;
+import uk.gov.companieshouse.api.model.accounts.smallfull.creditorswithinoneyear.CurrentPeriod;
+import uk.gov.companieshouse.api.model.accounts.smallfull.creditorswithinoneyear.PreviousPeriod;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.AccrualsAndDeferredIncome;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.BankLoansAndOverdrafts;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.CreditorsWithinOneYear;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.FinanceLeasesAndHirePurchaseContracts;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.OtherCreditors;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.TaxationAndSocialSecurity;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.Total;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorswithinoneyear.TradeCreditors;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.CreditorsWithinOneYearTransformer;
+
+@Component
+public class CreditorsWithinOneYearTransformerImpl implements CreditorsWithinOneYearTransformer {
+
+  @Override
+  public CreditorsWithinOneYear getCreditorsWithinOneYear(
+      CreditorsWithinOneYearApi creditorsWithinOneYearApi) {
+
+    CreditorsWithinOneYear creditorsWithinOneYear = new CreditorsWithinOneYear();
+
+    AccrualsAndDeferredIncome accrualsAndDeferredIncome = new AccrualsAndDeferredIncome();
```

Comment:
There's no need to instantiate nested models if `creditorsWithinOneYearApi` is null; Thymeleaf handles that for you